### ISSUE TITLE
arduino: 'save as' dialog crashes with 'arduino: No GSettings schemas are installed on the system'

### ISSUE THREAD

**Issue by raboof** [NixOS/nixpkgs#100137]
**Describe the bug**
The Arduino IDE works well for existing files, but when I try to save a new file or use 'Save as' the program crashes with:

```
(java:14796): GLib-GIO-ERROR **: 11:26:50.963: No GSettings schemas are installed on the system
/nix/store/p5bqkpq25vlj6m5ibm4ggraldr2lspdx-arduino-1.8.12/bin/arduino: line 35: 14796 Trace/breakpoint trap   (core dumped) "$JAVA" "${JAVA_OPTIONS[@]}" processing.app.Base "$@"
```

This is very similar to #21647

**To Reproduce**
1. open arduino
2. File->New
3. File->Save

**Metadata**

 - system: `"x86_64-linux"`
 - host os: `Linux 5.9.0-rc5, NixOS, 21.03.git.daaa0e33505 (Okapi)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.7`
 - channels(root): `"nixos-20.03pre194293.2436c27541b"`
 - nixpkgs: `/home/aengelen/nixpkgs`

**Comment by worldofpeace** [comment]
Please see the gnome section of the nixpkgs manual. There's an FAQ with that error message.

**Comment by raboof** [comment]
this has since been fixed on nixos-unstable

**Comment by raboof** [comment]
not sure what I was thinking above, it's still there on nixos-unstable (733e537a8ad76fd355b6f501127f7d0eb8861775)

**Comment by raboof** [comment]
> Please see the gnome section of the nixpkgs manual. There's an FAQ with that error message.

Right, https://nixos.org/manual/nixpkgs/unstable/#ssec-gnome-packaging . I'll look into that a bit.

**Comment by Emantor** [comment]
Fixed in #108216
